contracts/double-dice-roll/src/contract.rs

```diff
@@ -29,11 +29,27 @@ pub fn instantiate(
     set_contract_version(deps.storage, CONTRACT_NAME, CONTRACT_VERSION)?;
     NOIS_PROXY.save(deps.storage, &nois_proxy_addr)?;
 
+    set_contract_version(
```

Comment:
Since you add this here you can remove line 29 and the constants `CONTRACT_NAME` and `CONTRACT_VERSION`

Reply:
fixed on the lotto PR